setup.py

```diff
@@ -1,9 +1,11 @@
-from setuptools import find_packages, setup
-from jax_unirep import __version__
+from setuptools import setup
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+with open("jax_unirep/version.py") as fp:
+    version = fp.read().split('"')[1]
```

Comment:
Nice! I didn't know about this practice beforehand. Thanks for helping us with this!